### ISSUE TITLE
An example wouldn't hurt at all

### ISSUE THREAD

**Issue by weitzhandler** [dotnet/docs#8972]
Please implement an example to `ValueTask` and `ValueTask&lt;T&gt;`.

Thank you!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d85822ec-73d6-6963-1ea0-9cbfe01b791f
* Version Independent ID: 2afd17e0-891a-b631-02fc-b391d61d0e38
* Content: [ValueTask Struct (System.Threading.Tasks)](https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask?view=netcore-2.1)
* Content Source: [xml/System.Threading.Tasks/ValueTask.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Threading.Tasks/ValueTask.xml)
* Product: **dotnet-api**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by pkulikov** [comment]
@weitzhandler while the docs are not updated, the following blog post might be useful:
https://blogs.msdn.microsoft.com/dotnet/2018/11/07/understanding-the-whys-whats-and-whens-of-valuetask/

**Comment by weitzhandler** [comment]
Yeah I read that post today that's what led me to the article in first place...
But thanks for linking it in. Very worthwhile indeed.
